test/lib/mayaUsd/render/mayaToHydra/cpp/testUsdPointInstancePicking.cpp

```diff
@@ -65,9 +65,6 @@ TEST(TestUsdPointInstancePicking, pickPointInstance)
     mouseClick(Qt::MouseButton::LeftButton, active3dView.widget(), primMouseCoords);
     active3dView.refresh();
 
-    // When picking on the boundary of multiple objects, one Hydra pick hit per
-    // object is returned.  Therefore test that the expected selected path is
-    // in the selection.
-    ASSERT_GE(sn->size(), 1u);
+    ASSERT_EQ(sn->size(), 1u);
```

Comment:
Adjusted since we should no longer have more than one prim selected when single-clicking.